src/dlgRoomExits.cpp

```diff
@@ -2286,13 +2302,14 @@ void dlgRoomExits::slot_checkModified()
                                            ? 1 : 0 ;
                     currentExit.weight = pI->text(2).toInt();
                     currentExit.hasStub = false;
-                    if( foundMap.contains( currentCmd )
-                        && foundMap.value( currentCmd )->destination == currentExit.destination
-                        && foundMap.value( currentCmd )->door        == currentExit.door
-                        && foundMap.value( currentCmd )->hasNoRoute  == currentExit.hasNoRoute
-                        && foundMap.value( currentCmd )->weight      == currentExit.weight      )
+                    auto exit = foundMap.value(currentCmd);
```

Comment:
Not sure I follow, do you mean indenting the _continuation_ lines of the conditions by a further level?

            if (exit
                    && exit->destination == currentExit.destination
                    && exit->door        == currentExit.door
                    && exit->hasNoRoute  == currentExit.hasNoRoute
                    && exit->weight      == currentExit.weight      ) {
                foundMap.remove( currentCmd );
            } else {
                isModified = true;
                break;
            }